tests/tensorflow/r2.13/common.libsonnet

```diff
@@ -0,0 +1,327 @@
+          },
+          'tpu-version': {
+            image: 'google/cloud-sdk',
+            command: null,
+          },
+        },
+      },
+    },
+  },
+  TfVisionTest:: self.ModelGardenTest + common.TfNlpVisionMixin {
+    scriptConfig+: {
+      runnerPath: 'official/vision/train.py',
+    },
+  },
+  TfNlpTest:: self.ModelGardenTest + common.TfNlpVisionMixin {
+    scriptConfig+: {
+      runnerPath: 'official/nlp/train.py',
+    },
+  },
+  TfRankingTest:: self.ModelGardenTest {
+    paramsOverride:: {
+      runtime: {
+        distribution_strategy: error 'Must set `runtime.distribution_strategy`',
+      },
+      task: {
+        train_data: {
+          input_path: '$(CRITEO_DATA_DIR)/train/*',
+          global_batch_size: 16384,
+        },
+        validation_data: {
+          input_path: '$(CRITEO_DATA_DIR)/eval/*',
+          global_batch_size: 16384,
+        },
+        model: {
+          num_dense_features: 13,
+          bottom_mlp: [512, 256, 64],
+          embedding_dim: 64,
+          top_mlp: [1024, 1024, 512, 256, 1],
+          vocab_sizes: [
+            39884406,
+            39043,
+            17289,
+            7420,
+            20263,
+            3,
+            7120,
+            1543,
+            63,
+            38532951,
+            2953546,
+            403346,
+            10,
+            2208,
+            11938,
+            155,
+            4,
+            976,
+            14,
+            39979771,
+            25641295,
+            39664984,
+            585935,
+            12972,
+            108,
+            36,
+          ],
+        },
+      },
+      trainer: {
+        use_orbit: true,
+        validation_interval: 90000,
+        checkpoint_interval: 270000,
+        validation_steps: 5440,
+        train_steps: 256054,
+        optimizer_config: {
+          embedding_optimizer: 'SGD',
+          lr_config: {
+            decay_exp: 1.6,
+            decay_start_steps: 150000,
+            decay_steps: 136054,
+            learning_rate: 30,
+            warmup_steps: 8000,
+          },
+        },
+      },
+    },
+    command: [
+      'python3',
+      'official/recommendation/ranking/train.py',
+      '--params_override=%s' % (std.manifestYamlDoc(self.paramsOverride) + '\n'),
+      '--model_dir=$(MODEL_DIR)',
+    ],
+  },
+  imagenet:: {
+    scriptConfig+: {
+      trainFilePattern: '$(IMAGENET_DIR)/train*',
+      evalFilePattern: '$(IMAGENET_DIR)/valid*',
+    },
+  },
+  coco:: {
+    scriptConfig+: {
+      trainFilePattern: '$(COCO_DIR)/train*',
+      evalFilePattern: '$(COCO_DIR)/val*',
+      paramsOverride+: {
+        task+: {
+          annotation_file: '$(COCO_DIR)/instances_val2017.json',
+        },
+      },
+    },
+  },
+  local functional_schedule = '0 9 * * *',
+  Functional:: mixins.Functional {
+    schedule: if !(self.accelerator.type == 'tpu') || self.accelerator.name == 'v3-8' || self.accelerator.name == 'v4-8' then
```

Comment:
Can you check this logic? Does all the tests include based on this schedule?

Reply:
I don't think it's in the r2.12 version but it's here in nightly: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/tensorflow/nightly/common.libsonnet#L276

Reply:
Could you please use the one in r2.12